app/src/main/kotlin/com/simplemobiletools/flashlight/screens/MainScreen.kt

```diff
@@ -0,0 +1,245 @@
+    onFlashlightPress: () -> Unit,
+    showBrightDisplayButton: Boolean,
+    onBrightDisplayPress: () -> Unit,
+    showSosButton: Boolean,
+    sosActive: Boolean,
+    onSosButtonPress: () -> Unit,
+    showStroboscopeButton: Boolean,
+    stroboscopeActive: Boolean,
+    onStroboscopeButtonPress: () -> Unit,
+    showBrightnessBar: Boolean,
+    brightnessBarValue: Float,
+    onBrightnessBarValueChange: (Float) -> Unit,
+    showStroboscopeBar: Boolean,
+    stroboscopeBarValue: Float,
+    onStroboscopeBarValueChange: (Float) -> Unit,
+    showMoreApps: Boolean,
+    openSettings: () -> Unit,
+    openAbout: () -> Unit,
+    openSleepTimer: () -> Unit,
+    moreAppsFromUs: () -> Unit,
+) {
+    AdjustNavigationBarColors()
+    SettingsLazyScaffold(
+        customTopBar = { scrolledColor: Color, _: MutableInteractionSource, scrollBehavior: TopAppBarScrollBehavior, statusBarColor: Int, colorTransitionFraction: Float, contrastColor: Color ->
+            TopAppBar(
+                title = {},
+                actions = {
+                    val actionMenus = remember {
+                        val settings =
+                            ActionItem(R.string.settings, icon = Icons.Filled.Settings, doAction = openSettings, overflowMode = OverflowMode.NEVER_OVERFLOW)
+                        val about = ActionItem(R.string.about, icon = Icons.Outlined.Info, doAction = openAbout, overflowMode = OverflowMode.NEVER_OVERFLOW)
+                        val sleepTimer = ActionItem(R.string.sleep_timer, doAction = openSleepTimer, overflowMode = OverflowMode.ALWAYS_OVERFLOW)
+                        val list = mutableListOf(settings, about, sleepTimer)
+                        if (showMoreApps) {
+                            list += ActionItem(R.string.more_apps_from_us, doAction = moreAppsFromUs, overflowMode = OverflowMode.ALWAYS_OVERFLOW)
+                        }
+                        list.toImmutableList()
+                    }
+                    var isMenuVisible by remember { mutableStateOf(false) }
+                    ActionMenu(
+                        items = actionMenus,
+                        numIcons = 2,
+                        isMenuVisible = isMenuVisible,
+                        onMenuToggle = { isMenuVisible = it },
+                        iconsColor = scrolledColor
+                    )
+                },
+                scrollBehavior = scrollBehavior,
+                colors = topAppBarColors(statusBarColor, colorTransitionFraction, contrastColor),
+                modifier = Modifier.topAppBarPaddings(),
+                windowInsets = topAppBarInsets()
+            )
+        }) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.SpaceEvenly
+        ) {
+            Icon(
+                modifier = Modifier
+                    .size(dimensionResource(id = R.dimen.main_button_size))
```

Comment:
Don't use `R.dimen` rather create dimensions in a separate file

Reply:
Should we provide it in a similar way to colors, to make sure we use similar dimensions across the apps?

Reply:
yes